lib/api/src/grpc/conversions.rs

```diff
@@ -337,6 +337,7 @@ impl From<QuantizationSearchParams> for segment::types::QuantizationSearchParams
             rescore: params
                 .rescore
                 .unwrap_or(default_quantization_rescore_value()),
+            oversampling: None, // TODO: Add `oversampling` to gRPC!
```

Comment:
__TODO__

Reply:
Why is oversampling not propagated in this direction?